cockatrice/src/pictureloader.cpp

```diff
@@ -384,10 +384,11 @@ void PictureLoaderWorker::startNextPicDownload()
         picDownloadFailed();
     } else {
         QUrl url(picUrl);
-        qDebug() << "PictureLoader: [card: " << cardBeingDownloaded.getCard()->getCorrectedName()
-                 << " set: " << cardBeingDownloaded.getSetName()
-                 << QString("]: Trying to download picture from %1:").arg(picDownload ? "url" : "cache")
-                 << url.toDisplayString();
+        qDebug() << qPrintable(QString("PictureLoader: [card: %1 set: %2]: Trying to %3: %4")
```

Comment:
what's the rationale behind using qPrintable here? converting the qstring to a char * isn't necessary for qDebug

Reply:
Printing `QString` directly adds quotes and escaping when printing, which I find quite ugly especially since I was using `.arg()`, so I figured I'd remove while I was there. I don't really care either way (and apparently the right thing to use would be qUtf8Printable and not qPrintable). Reverting…